docs/guide/troubleshooting.md

```diff
@@ -43,6 +43,14 @@ To solve this:
   $ sudo sysctl fs.inotify.max_user_instances=8192
   $ sudo sysctl fs.inotify.max_user_watches=524288
   ```
+#### Alternatively try Modifying Configs
```

Comment:
Probably only the first word should start with a capital letter for consistency with the other sections. I'd suggest an alternate heading 
```suggestion
#### Modify `DefaultLimitNOFILE`
```

Reply:
I'm not suggesting to make the new steps required - just questioning if they need their own heading. You could still start the text with "If the above steps don't work then..."